images/dind/Dockerfile

```diff
@@ -9,11 +9,22 @@ FROM fedora:23
 ## Configure systemd to run in a container
 ENV container=docker
 
-RUN systemctl mask systemd-remount-fs.service dev-hugepages.mount \
-  sys-fs-fuse-connections.mount systemd-logind.service getty.target \
-  console-getty.service dnf-makecache.service lvm2-lvmetad.service \
-  systemd-udevd.service systemd-udev-hwdb-update.service \
-  systemd-udev-trigger.service docker-storage-setup.service
+RUN systemctl mask\
+ auditd.service\
+ console-getty.service\
+ dev-hugepages.mount\
+ dnf-makecache.service\
+ docker-storage-setup.service\
+ getty.target\
+ lvm2-lvmetad.service\
+ sys-fs-fuse-connections.mount\
+ systemd-logind.service\
+ systemd-remount-fs.service\
+ systemd-udev-hwdb-update.service\
+ systemd-udev-trigger.service\
+ systemd-udevd.service\
+ systemd-vconsole-setup.service
```

Comment:
Yes, I periodically mask out processes that fail to start and that dind doesn't depend on.